Examples/minimum_user_for_s2e/src/src_user/IfWrapper/Sils/sils_sci_if.cpp

```diff
@@ -0,0 +1,180 @@
+#pragma section REPRO
+/**
+ * @file
+ * @brief   sils_sci_if
+ * @details SCI COMでやりとりするIF
+ */
+
+#include "sils_sci_if.hpp"
+
+#ifdef WIN32
+SCIComPort::SCIComPort(int port)
+{
+  char port_settings[15];
```

Comment:
[IMO]

```suggestion
  char port_name[15];
```

とか？（settingsではなさそう）

unix側も

Reply:
port_nameにしました